crates/flowctl/src/auth/mod.rs

```diff
@@ -47,6 +47,21 @@ pub enum Command {
     /// Unlike 'read' or 'write', the subject of an 'admin' grant also inherits
     /// capabilities granted to the object role from still-other roles.
     Roles(roles::Roles),
+
+    /// Fetches and prints an auth token that can be used to access a Flow data plane.
```

Comment:
Are the changes in this file related to connector networking? At first glance they seem unrelated and maybe not intentional.

Reply:
Yeah, this is related and intentional. See [this commit message](https://github.com/estuary/flow/pull/893/commits/86cd537ed02e0ce5e0d832b64c891593f1a28d56) for rationale.

Reply:
Ahhh I see, thanks!